test/README.md

```diff
@@ -0,0 +1,104 @@
+# Cockpit-Composer Integration Test
+
+The integration test for Cockpit Composer! It is performed on the
+application level and tests whether the business requirements are met
+regardless of app internal architecture, dependencies, data integrity and such.
+Actually we need to follow the end-user flows and assert they get the intended
+experience and focus on the behavior of the thing as the user would see it.
+
+The integration tests are powered by [Cockpit test
+framework](https://github.com/cockpit-project/cockpit/tree/master/test), which is simple and easy to
+debug for test development.
+
+## Requirement
+
+For testing, the following dependencies are required:
+
+    $ sudo dnf install curl expect xz rpm-build chromium-headless \
+        libvirt-daemon-kvm libvirt-client python3-libvirt
+
+And `chrome-remote-interface` and `sizzle` Javascript librarys need to be installed:
+
+    $ npm install
+
+## Introduction
+
+Before running the tests, ensure Cockpit-Composer environment has been built:
+
+    $ make vm
+
+To run all tests run the following:
+
+    $ ./test/verify/run-tests
+
+Alternatively you can run an individual test like this:
+
+    $ ./test/verify/check-image
+
+To see more verbose output from the test, use the `--verbose` and/or `--trace` flags:
+
+    $ ./test/verify/check-image --verbose --trace
+
+In addition if you specify `--sit`, then the test will wait on failure and allow you to log into
+cockpit and/or the test instance and diagnose the issue. An address will be printed of the test
+instance.
+
+    $ ./test/verify/check-image --trace --sit
+
+Normally each test starts its own chromium headless browser process on a separate random port. To
+interactively follow what a test is doing, start the browser manually and tell the test which debug
+port it should attach to:
+
+    $ chromium-browser --remote-debugging-port=9222 about:blank
+    $ TEST_CDP_PORT=9222 ./test/verify/check-image --trace
```

Comment:
This is correct, but I would advise to use `TEST_SHOW_BROWSER` as you need to make sure, you have correctly set up the browser (and also that HOME is clean enough).